documentation/BookAbstracts.json

```diff
@@ -70,25 +70,29 @@
     "reference" : {
       "type" : "Reference",
       "publicationContext" : {
-        "type" : "Journal",
-        "title" : "Tim's lovely publishing house",
-        "printIssn" : "2222-2227",
-        "onlineIssn" : "1111-1119"
+        "type" : "Book",
```

Comment:
Journal is now Book?

Reply:
It shouldn't have been Journal to start off with :D